template/src/project_name/accounts/models.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib.auth.models import AbstractUser
+
+class CustomUser(AbstractUser):
+    pass
+
+    def __str__(self):
+        return self.username
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Please ensure that your IDE it set to ensure there is a trailing line ending. The little red symbol above represents that there is not line end on the file line.‏